templates/common/azure/files/azure-chrony.yaml

```diff
@@ -0,0 +1,50 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/chrony.conf"
+contents:
+  inline: |
+    # This file is shipped by the Machine Config Operator to synchronize time RHCOS host time on azure.
+    # It uses PTP source clock, see https://docs.microsoft.com/en-us/azure/virtual-machines/linux/time-sync
+    #
+    # Use public servers from the pool.ntp.org project.
+    # Please consider joining the pool (http://www.pool.ntp.org/join.html).
+    pool 2.rhel.pool.ntp.org iburst
```

Comment:
Question: Since this doesn't work , do we want to comment this out?

Reply:
if it's not causing issue even if not working but that's something we do ship anyway, I'd leave it there